src/requests/requests.js

```diff
@@ -30,3 +31,19 @@ export const fetchMovies = async (searchQuery, selectedSortOption, selectedGenre
     }
   }
 };
+
+export const fetchMovie = async ({ params }) => {
+  const response = await axios.get(`http://localhost:4000/movies/${params.movieId}`);
```

Comment:
This is the second place you are using `"localhost:4000"`.
You can use something like `axios.create({baseURL: "http://localhost:4000"})` to avoid code duplication.
And the `baseUrl` should probably come from env variable.